src/components/application_manager/include/application_manager/commands/command_impl.h

```diff
@@ -169,6 +181,29 @@ class CommandImpl : public Command {
    */
   void ReplaceHMIByMobileAppId(smart_objects::SmartObject& message);
 
+  /**
+   * @brief Validate string syntax in smart object
+   * @param obj Smart object to check
+   * @return true if syntax if all object strings is correct otherwise false
```

Comment:
cannot understand what is
```return true if syntax if all object strings is correct otherwise false```
please update this and other similar comments

Reply:
@yang1070 sorry, there was a misprint. Description was updated in https://github.com/smartdevicelink/sdl_core/pull/1820/commits/74837cf41e96439eb131227c961405b3af5091bb